react/LocationSearch.tsx

```diff
@@ -27,11 +27,18 @@ const getAddresses = (searchTerm: string) => {
 
 interface LocationSearchProps {
   onSelectAddress?: (selectedAddress: string) => void
+  renderEngineLogo?: (
+    props: React.ImgHTMLAttributes<HTMLImageElement> & { className?: string }
+  ) => HTMLImageElement
 }
 
-const LocationSearch: React.FC<LocationSearchProps> = ({ onSelectAddress }) => {
+const LocationSearch: React.FC<LocationSearchProps> = ({
+  onSelectAddress,
+  renderEngineLogo,
+}) => {
   const [searchTerm, setSearchTerm] = useState<string>('')
   const addresses = useMemo(() => getAddresses(searchTerm), [searchTerm])
+  const inputWrapperEl = useRef<HTMLDivElement>(null)
```

Comment:
nit: we usually name these ref variables with a `Ref` suffix
```suggestion
  const inputWrapperRef = useRef<HTMLDivElement>(null)
```